firebase-config/bandwagoner/src/main/java/com/googletest/firebase/remoteconfig/bandwagoner/ApiFragment.java

```diff
@@ -64,8 +64,8 @@ public void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
 
     frc = FirebaseRemoteConfig.getInstance();
-    frc.setConfigSettings(
-        new FirebaseRemoteConfigSettings.Builder().setDeveloperModeEnabled(true).build());
+    frc.setConfigSettingsAsync(
+        new FirebaseRemoteConfigSettings.Builder().setFetchTimeoutInSeconds(0L).build());
```

Comment:
Dumb question: IIUC, this test app calls a live RC endpoint, in which case setting a timeout of 0s seems aggressive; any chance the intent is to bypass the cache (`setMinimumFetchIntervalInSeconds `), which is closer to the functionality of developer mode, rather than setting a timeout?

Reply:
Not a dumb question! That is actually what I intended to do, but I set the wrong setting! Thanks for questioning it. :P